src/panels/lovelace/ha-panel-lovelace.ts

```diff
@@ -107,6 +113,13 @@ class LovelacePanel extends LitElement {
 
   public firstUpdated() {
     this._fetchConfig(false);
+    subscribeLovelaceUpdates(this.hass!.connection, () => {
```

Comment:
store the unsub function that is returned. When do you want to unsub ? If you don't want to unsub, add a comment.